services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/S3Publisher.java

```diff
@@ -69,7 +69,6 @@ public S3Publisher(ObjectStoreAccess objectStoreAccess, FailedObjectStoreOperati
    * be controlled through the configuration parameter <code>DistributionServiceConfig.forceUpdateKeyFiles</code>
    *
    * @param root The path of the directory that shall be published.
-   * @see Github issue #650
```

Comment:
@emmetsap - OK, `@see` doesn't work... but we should keep the information/reference to the particular github issue somewhere else

Reply:
I've now moved the reference below the description